docs/couchdb-quirks.md

```diff
@@ -34,6 +34,24 @@ reasons. In general, you can follow these two rules of thumb:
    omit the `sort` operator on the query (except if you want the `descending`
    order).
 
+### Warnings for slow requests
+
+When requesting a mango index, CouchDB can use an index. But there are also
+cases where no index can be used, or where the index is not optimal. Let's
+see the different scenarios:
+
+- CouchDB does use an index, it will respond with a warning, and cozy-stack
```

Comment:
I'm not sure I understand this: if CouchDB does use an index, why would we have a warning? Do you have an example of such situation?

Reply:
It was a typo